test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -160,9 +160,9 @@ def run(*ex, **kwargs):
     "test_empty2_dynamic_shapes": TestFailure(("cpu", "cuda")),
     "test_empty_strided_dynamic_shapes": TestFailure(("cpu", "cuda")),
     "test_index3_dynamic_shapes": TestFailure(("cpu", "cuda")),
-    "test_inductor_bucketize_dynamic_shapes": TestFailure("cpu"),
-    "test_inductor_bucketize_default_kwargs_dynamic_shapes": TestFailure("cpu"),
-    "test_inductor_bucketize_int_dynamic_shapes": TestFailure("cpu"),
+    "test_bucketize_dynamic_shapes": TestFailure("cpu"),
+    "test_bucketize_default_kwargs_dynamic_shapes": TestFailure("cpu"),
+    "test_bucketize_int_dynamic_shapes": TestFailure("cpu"),
```

Comment:
I wonder, why we don't need to mention `test_bucketize_add_autotune` and `test_bucketize_computed_offsets` here. Are those tests not attempted to run on CPU?

Reply:
The add/subtract gets compiled in that case so it ends up passing these tests. IMO it's fine to leave them un-skipped, even if they're passing for kind of the wrong reason